docs/messaging/messaging-metrics.md

```diff
@@ -18,6 +18,13 @@
 
 <!-- tocstop -->
 
+> **Warning**
+> Existing messaging instrumentations that are using
+> [v1.24.0 of this document](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/messaging/messaging-metrics.md)
+> (or prior) SHOULD NOT change the version of the messaging conventions that they emit
+> until a transition plan to the (future) stable semantic conventions has been published.
```

Comment:
FYI - 

One thing we'll be asking (as we learn from HTTP).

1. Can you create a central registry where instrumentation can link their transition plans?  e.g. somewhere on OpenTelemetry.IO.
2. (When Ready) Can you open tickets against all instrumentation repositories denoting the transition and actions we'd like them to take?

Note: this is a non-blocking comment.

Reply:
For (1) @trask is leading the way and we'll piggy-back on what he comes up with for HTTP and DB.

- For (2), I added it as an item to our [checklist for declaring stability](https://github.com/open-telemetry/semantic-conventions/issues/646).